src/main/scala/com/atomist/tree/pathexpression/PathExpressionParser.scala

```diff
@@ -17,8 +17,13 @@ trait PathExpressionParser extends CommonTypesParser {
 
   private def objectType: Parser[String] = identifierRefString(Set(), ident)
 
-  private def child: Parser[AxisSpecifier] = opt("child::") ^^
-    (s => Child)
+  private def child: Parser[AxisSpecifier] = opt("child::" | "/") ^^ {
```

Comment:
Do we need the `| "/"`? A second forward slash is just for descendants, or am I missing something?